spec/functional/mixin/from_file_spec.rb

```diff
@@ -21,6 +21,7 @@
   REAL_DATA = File.join(CHEF_SPEC_DATA, "mixin", "real_data.rb")
   INVALID_DATA = File.join(CHEF_SPEC_DATA, "mixin", "invalid_data.rb")
   NO_DATA = File.join(CHEF_SPEC_DATA, "mixin", "non_existant_data.rb")
+  DIRECTORY = File.expand_path("")
```

Comment:
Huh. I've never even though of doing that. I always use `"."`. Thank you for teaching me something!

This should be the same as `Dir.pwd` which might be a bit clearer. This is just a comment tho... not a blocking request for change.